string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,15 @@
+/* exported isVowel */
+
+// check if character is a lowercase vowel (a, e, i, o OR u)
+// check if character is an UPPERCASE vowel (A, E, I, O, U)
+// say true if it is
+// say false if it is not
+
+function isVowel(char) {
+  if (char === 'a' || char === 'e' || char === 'i' || char === 'o' || char === 'u') {
+    return true;
+  } else if (char === 'A' || char === 'E' || char === 'I' || char === 'O' || char === 'U') {
```

Comment:
Instead of making these two separate conditions, you can include all those expressions in the first condition. Orrrrr, you can just upper case or lowercase the letter before the conditional, so you only need to check one set 🤓